src/Core/Settings.cpp

```diff
@@ -1297,6 +1297,12 @@ The cache of uncompressed blocks stores data extracted for queries. ClickHouse u
 Possible values:
 
 - Any positive integer.
+)", 0) \
+DECLARE(Bool, merge_tree_use_deserialization_prefixes_cache, true, R"(
+Enables caching of deserialization prefixes during reading from Wide parts in MergeTree.
+)", 0) \
+DECLARE(UInt64Auto, merge_tree_prefixes_deserialization_thread_pool_size, "auto", R"(
+The number of threads used in a thread pool for parallel prefixes reading in Wide parts in MergeTree.
 )", 0) \
```

Comment:
We can clarify what 'prefixes' means in this context, as it is not widely used in our documentation, I believe